src/lib/sortPoints.test.js

```diff
@@ -0,0 +1,28 @@
+//verify that sortPoints correctly sorts a valid array of points.
+
+const sortPoints = require('./sortPoints')
+
+test('sortPoints correctly sorts list of (x,y) ints', function() {
+        //arrange
+        const array = [{x:12, y:3}, {x:4, y:7}, {x:6, y:12}];//input array
+        const expectedResult = [{x:4, y:7},{x:6, y:12},{x:12, y:3}];//what the array should be after sorting
+
+        //act
+        const result = sortPoints(array);//call function with array param
+
+        //assert
+        expect(result).toStrictEqual(expectedResult)//expect the result to be equal to the expected result
```

Comment:
what is stricktequal??

Reply:
i was using .toBe and it gave me an error and suggested i use this haha. I think it just means its a strict match. so the string, result == expected_result